diesel_tests/tests/types.rs

```diff
@@ -383,10 +383,10 @@ fn timestamp_from_sql() {
     use diesel::data_types::PgTimestamp;
 
     let query = "'2015-11-13 13:26:48.041057-07'::timestamp";
-    let expected_value = PgTimestamp(500736408041057);
+    let expected_value = PgTimestamp(500_736_408_041_057);
```

Comment:
This line, and others in the same file, were not caught by `bin/test`. I changed them anyway, because it seems like they should have been caught by `bin/test`, and I assume future development might make `bin/test` catch them.